tutor/src/components/cc-dashboard/help.cjsx

```diff
@@ -28,28 +28,48 @@ CCDashboardHelp = React.createClass
         <p className="fine-print">This guide is always accessible from your main menu.</p>
         <div className="side-by-side">
           <div className="help">
-            <h3 className="improvements">
-              In January 2017, you’ll see the following improvements to Concept Coach:
-            </h3>
-
+            <b>
+              Add <CourseGroupingLabel plural {...glprops} /> to your course.
```

Comment:
`CourseGroupingLabel` is used here, should other instances of "section" in this welcome also be using the label?  are there really going to be coach courses that are high school though?